Misc/NEWS.d/next/Library/2019-12-07-16-32-42.bpo-38979.q0sIHy.rst

```diff
@@ -0,0 +1,3 @@
+return class from __class_getitem__ to simplify subclassing. E.g. `clss =
```

Comment:
You might want to use double backtick for `__class_getitem__`

Reply:
I agree with both @isidentical comments.  @amiremohamadi please address them.
Otherwise, the PR looks awesome.